README.md

```diff
@@ -1,20 +1,65 @@
+<h2 align="center" >A chat interface that can control your product </h3>
+<p align="center"></p>
 
-E.g. a CRM user could ask: "What's the status of the B Corp deal?". A product analytics tool user could ask: "What did users think of the recent UI change?".
+<p align="center">🌐 <a href="https://superflows.ai">Website</a> · 📚 <a href="https://docs.superflows.ai">Docs</a> ·  💬 <a href="https://join.slack.com/t/superflowsusers/shared_invite/zt-1zf93teqq-0I9H_niiGruVDPFVSzGj9w">Slack</a></p>
+<br />
 
 https://github.com/Superflows-AI/superflows/assets/33871096/77c463c1-5e06-45a8-a32a-9f1fd8af0dac
 
+# Superflows
+
+Superflows makes it easy to add an AI assistant to a software product. This lets users ask questions in natural language and the assistant makes API calls to answer them.
+
+E.g. a CRM user could ask: "What's the status of the B Corp deal?". A product analytics tool user could ask: "What did users think of the recent UI change?". These questions would be addressed by fetching information from your product's API and using an LLM to generate the answer.
+
 You can try out the cloud version for free [**here**](https://dashboard.superflows.ai) or self-host. You can learn more on our [documentation pages](https://docs.superflows.ai/).
 
+### Readme nav
+
+[Features](#features) | [Setup](#setup) | [Development](#development)
+
+
 ## Features
 
 - Developer dashboard to configure and test your product assistant
 - Public streaming API
 - Answers in the same language the question is asked in
 - UI components (find React components [here](https://github.com/Superflows-AI/chat-ui))
 
+### Dashboard
+
+Superflows has a testing dashboard where you can configure, evaluate and debug your AI assistant before putting it into production. 
+
+You can interact with your assistant on the 'Playground'. 'Developer mode' shows you under the hood of what the assistant is thinking and planning. 'Mock API responses' generates fake data, enabling you to check the behaviour of the assistant without connecting it to an API. 
+
+<img src="public/crm-ai.png" width="500px" alt="Superflows Logo" />
+
+The 'Actions' page lets you control which API endpoints your assistant can call to answer user queries in a simple dashboard.
+
+These endpoints can easily be uploaded via an Open API Specification, or entered manually. 
+
+<img src="public/sf-actions.png" width="500px" alt="Superflows Logo" />
+
+
+### API
+
+Our API specification can be found at on our [documentation page](https://docs.superflows.ai/docs/category/api-specification). 
+
+### UI Components
+
+Superflows comes with out-of-the-box React UI components. These components let you integrate Superflows into your product in a few lines of code ([integration guide](https://docs.superflows.ai/docs/integration-guide/react)). 
+
+<img src="public/superflows-sidebar.gif" width="500px" alt="Superflows Logo" />
+
+
+
 ## Setup
```

Comment:
I would move this above Features since it's so short and this is pretty important imo